example/full/instance.properties

```diff
@@ -378,6 +378,48 @@ sleeper.bulk.import.emr.ebs.volume.type=gp2
 # This can be a number from 1 to 25.
 sleeper.bulk.import.emr.ebs.volumes.per.instance=4
 
+# The class to use to perform the bulk import.
+sleeper.bulk.import.emr.serverless.class.name=sleeper.bulkimport.job.runner.dataframelocalsort.BulkImportDataframeLocalSortDriver
+
+# The architecture for EMR Serverless to use. X86_64 or ARM (Coming soon)
+sleeper.bulk.import.emr.serverless.architecture=X86_64
+
+# The switch to enable EMR Serverless over persistent EMR.
+sleeper.bulk.import.emr.serverless.enabled=true
```

Comment:
This property is for BULK_IMPORT_EMR_SERVERLESS_ENABLED which appears not to be used.

Reply:
It is no longer needed and has been removed